examples/esp-todomvc-react/js/footer.jsx

```diff
@@ -0,0 +1,64 @@
+/*global React*/
+var app = app || {};
+
+(function () {
+    "use strict";
+
+    app.Footer = React.createClass({
+
+        handleClear: function () {
+            this.props.router.publishEvent("todoList", "clearCompleted", {});
+        },
+
+        render: function () {
+            var footer = this.props.model;
+            if (!footer.visible) {
+                return null;
```

Comment:
This was copied from the React TodoMVC example.

https://facebook.github.io/react/docs/component-specs.html says regarding `render()`:

> You can also return null or false to indicate that you don't want anything rendered.